src/types.js

```diff
@@ -62,6 +62,17 @@ export function isObject(value) {
 }
 
 /**
+ * Determines if value is actually a number.
+ * NaN is not considered a number in this function.
+ * String numbers are not considered numbers in this function.
+ * @param {*} value
+ * @return {boolean}
+ */
+export function isNumber(value) {
```

Comment:
Changed to `isFiniteNumber` and the implementation changed to be same as native.